.gitlab-ci-full-integration.yml

```diff
@@ -5,6 +5,9 @@ test:integration:
   stage: test
   tags:
     - mender-qa-worker-integration-tests
+  timeout: 24h
```

Comment:
Okokok :upside_down_face: 
I was thinking about running these test on a tiny machine that can hum and sing integration tests all night :notes:
Still, I don't think it will take 24h - I'll adjust.